pyccel/ast/core.py

```diff
@@ -3573,6 +3564,7 @@ def rename(self, newname):
                              self.body, cls_name=self.cls_name)
 
 
+# TODO: [EB 06.01.2021] Is this class used? What for?
```

Comment:
Have you searched for it in the whole project with `grep`?

Reply:
Sorry I meant to go back to this. It seems to be associated with the `@python` decorator but I am not certain what that decorator does. The inheritance from `FunctionDef` probably explains why it only appears in 5 lines excluding the contents of the class

Reply:
@saidctb Do you know what the `@python` decorator was for? Maybe for functions that should not be translated to Fortran/C?

Reply:
I suggest we create a dedicated issue where we say that the `@python` decorator should be tested, and then either fix it or remove it.